File: AvailabilityCollector/Models/Razpolozljivost.cs
namespace AvailabilityCollector.Models;

public class Razpolozljivost
{
    public int ID { get; set; }
    public string RazpolozljivostJSON { get; set; } = "";
    public string MesecLeto { get; set; } = "";
    public string Type { get; set; } = "";  
    public int? ZaporedniTeden { get; set; }

    public int WorkerID { get; set; }
    public Worker? Worker { get; set; }
}

File: Android/app/src/main/java/Android/availibityCollector/data/models/Razpolozljivost.kt
package Android.availibityCollector.data.models

import com.google.gson.annotations.SerializedName

data class Razpolozljivost(
    @SerializedName("id") val id: Int,
    @SerializedName("razpolozljivostJSON") val razpolozljivostJSON: String,
    @SerializedName("mesecLeto") val mesecLeto: String,
    @SerializedName("type") val type: String,
    @SerializedName("zaporedniTeden") val zaporedniTeden: Int?,
    @SerializedName("workerID") val workerId: Int
)

data class RazpolozljivostCreate(
    @SerializedName("razpolozljivostJSON") val razpolozljivostJSON: String,
    @SerializedName("mesecLeto") val mesecLeto: String,
    @SerializedName("type") val type: String,
    @SerializedName("zaporedniTeden") val zaporedniTeden: Int?,
    @SerializedName("workerID") val workerId: Int
)
